test/known_issues/test-fs-copyfile-respect-permissions.js

```diff
@@ -39,8 +40,12 @@ function beforeEach() {
 // Test promises API.
 {
   const { source, dest, check } = beforeEach();
-  assert.throws(async () => { await fs.promises.copyFile(source, dest); },
-                check);
+  (async () => {
+    await assert.rejects(
+      async () => { await fs.promises.copyFile(source, dest); },
+      check
+    );
+  })();
```

Comment:
I think this can be simplified to:

```js
  assert.rejects(async () => { await fs.promises.copyFile(source, dest); },
                 check);
```

In other words, no need to wrap the `assert.rejects()` in an `async` function, I don't think?

Reply:
Although our docs show it wrapped like this, so maybe best to leave it...